packages/openbridge-webcomponents/src/navigation-instruments/readout-input/readout-input.css

```
@@ -0,0 +1,518 @@
+      .no-hug-content.type-regular
+    ) {
+      gap: 1px;
+    }
+
+    &.variant-value.medium .icon-container {
+      color: var(--element-neutral-color);
+    }
+  }
+}
+
+&.direction-horizontal.readout-style-stack.type-regular {
```

Stylelint (17.7.0)

[error] Missing scoping root (nesting-selector-no-missing-scoping-root)

(nesting-selector-no-missing-scoping-root)